internal/validation/comprehensive.go

```diff
@@ -0,0 +1,115 @@
+package validation
```

Comment:
Why do we need to use "internal" package ?
We don't use internal packages and IMO it's a bad way.